test/celeritas/optical/Cerenkov.test.cc

```diff
@@ -113,10 +114,9 @@ Span<double const> get_refractive_index()
     return make_span(refractive_index);
 }
 
-double convert_to_energy(double wavelength)
+real_type convert_to_energy(real_type wavelength)
```

Comment:
Isn't it better to just use `OpticalUtils::wavelength_to_energy` directly and drop this one?

Reply:
yeah I'll just do that.